eslint.config.js

```diff
@@ -14,11 +15,36 @@ export default [
     ...eslintConfigESLint,
     eslintConfigESLintFormatting,
     {
-        files: ["**/tests/lib/**"],
+        files: ["packages/espree/tests/lib/**", "packages/eslint-scope/tests/**"],
         languageOptions: {
             globals: {
                 ...globals.mocha
             }
+        },
+        plugins: {
```

Comment:
Did you intend to add more to this config?

Reply:
Yeah, this is to match the [eslint-scope ESLint config.](https://github.com/eslint/eslint-scope/blob/fa003eeb3e6a4a6b77c6911fd8ca053960c7d471/eslint.config.js#L20)

I merged both the configs, but, I think we can add different config for `packages/espree/tests/lib/**` & `packages/eslint-scope/tests/lib/**`

Reply:
> I think we can have different config for `packages/espree/tests/lib/**` & `packages/eslint-scope/tests/lib/**`

Sounds good to me to have different configs, because eslint-scope tests use `chai` while espree tests don't (I'm not sure why `chai` was listed in espree dev deps).

Reply:
Updated, also removed chai from espree's dev dep.